### PR TITLE
Link fixes per doc build report

### PR DESCRIPTION
**Pull request by mikehoffms**
AB#43804752

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ The following sections announce additional features available in Microsoft Edge
 
 The **Styles** pane now has better support for editing styles that were created with the [CSS Object Model (CSSOM)](https://drafts.csswg.org/cssom) APIs.  Many CSS-in-JS frameworks and libraries use the CSSOM APIs under the hood to construct styles.
 
-You can now edit styles added in JavaScript using [Constructable Stylesheets](https://wicg.github.io/construct-stylesheets/).  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
+You can now edit styles added in JavaScript using [Constructable Stylesheets](https://web.dev/constructable-stylesheets/).<!-- todo: https://wicg.github.io/construct-stylesheets/ is 404 -->  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md:145]:
```suggestion
You can now edit styles added in JavaScript using [Constructable Stylesheets](https://web.dev/constructable-stylesheets/).<!-- todo: https://wicg.github.io/construct-stylesheets/ is 404 -->  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```
@captainbrosset  - the problem with changing the 404 link to https://web.dev/constructable-stylesheets/ is that that page itself contains the 404 link

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md:145]:
```suggestion
You can now edit styles that were added in JavaScript using [the `CSSStyleSheet` interface](https://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface), which is a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```

File: microsoft-edge/devtools-guide-chromium/css/css-in-js.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ This feature is available starting with Microsoft Edge version 93. <!-- delete s
 
 The **Styles** pane supports editing styles that were created with the [CSS Object Model (CSSOM)](https://developer.mozilla.org/docs/Web/API/CSS_Object_Model) APIs.  Many CSS-in-JS frameworks and libraries use the CSS Object Model APIs under the hood to construct styles.
 
-You can edit styles added in JavaScript using [Constructable Stylesheets](https://wicg.github.io/construct-stylesheets/).  Constructable Stylesheets are a way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
+You can edit styles added in JavaScript using [Constructable Stylesheets](https://web.dev/constructable-stylesheets/).<!-- todo: https://wicg.github.io/construct-stylesheets/ is 404 -->  Constructable Stylesheets are a way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/css/css-in-js.md:55]: See my other comment below in the whats-new entry.

**Review by captainbrosset — APPROVED** [review]
Approving already, but I also made a suggestion for the constructable stylesheet link. See below.
Let me know when this is ready and I can merge it quickly.

File: microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md]: You can now edit styles that were added in JavaScript using the `CSSStyleSheet` interface, which is a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).  See [The `CSSStyleSheet` Interface](https://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface) in _CSS Object Model (CSSOM)_.

**Pull request merged**